packages/plugin-browserify/src/plugin.ts

```diff
@@ -65,11 +126,16 @@ export class SnapsBrowserifyTransform extends Transform {
       inputSourceMap,
     });
 
-    if (result) {
-      this.push(result.code);
-    }
-
-    callback();
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    postBundle(this.#options, result!.code)
```

Comment:
Are we sure `result` cannot be undefined in our case?

Reply:
`result` is `null` if the input code is `null`. I don't think that's ever the case here. We could probably just change the function to always return something? `code` accepting `null` is leftover from the previous implementation.